templates/frontends-pipeline-pr-checks/Jenkinsfile

```diff
@@ -55,19 +55,20 @@ pipeline {
                 stage('Unit Testing') {
                     agent { label 'insights' }
                     environment {
-                            IMG_TAG=sh(script: "git rev-parse --short=8 HEAD", returnStdout: true).trim()
-                            TEST_CONT="${PROJECT_NAME}-unit-tests-${IMG_TAG}"
+                        TEST_CONT="${PROJECT_NAME}-unit-tests-${IMG_TAG}"
                     }
                     when {
                         expression {
                             return (RUN_UNIT_TESTS_FLAG == 'true')
```

Comment:
where is `RUN_UNIT_TESTS_FLAG` and similar variables defined ?

Reply:
but these are Groovy variables (not environment variables, not pipeline parameters) - or are these JJB template configured variables are actually added as Groovy variables ? if so, then ignore my comment